app/services/the_tvdb_service.rb

```diff
@@ -0,0 +1,122 @@
+class TheTvdbService
+  BASE_URL = 'https://api.thetvdb.com'.freeze
+  API_KEY = ENV['THE_TVDB_API_KEY'].freeze
+
+  attr_accessor :media_id
+
+  def initialize
+    @token = api_token
```

Comment:
It's better to just have `api_token` return the same token if it's called again and then use that directly

Reply:
so something like this?
```
  def api_token
    return @token if @token.present?
```

Reply:
Pretty much, yep